app/models/user.rb

```diff
@@ -1,5 +1,6 @@
 class User < ActiveRecord::Base
   authenticates_with_sorcery!
+  before_create :create_organiser_token
```

Comment:
pending organizer will be the one that still has organiser_token - the one who clicked on the link won't have it anymore as you're clearing this field when confirming organizer right?

And I'm not sure I'm following question about required thing ;)